.changie.yaml

```
@@ -0,0 +1,18 @@
+changesDir: .changes
+unreleasedDir: unreleased
+changelogPath: CHANGELOG.md
+versionExt: md
+versionFormat: '## {{.Version}} ({{.Time.Format "January 02, 2006"}})'
+kindFormat: '### {{.Kind}}'
+changeFormat: '* {{.Body}}'
```

Nit: One fun option here is that we can have `changie` prompt automatically for an issue/PR number. An example of this can be seen in the [changie repository's configuration file](https://github.com/miniscruff/changie/blob/main/.changie.yaml).

I think it'd be something like this to match our current changelog formatting:

```suggestion
changeFormat: '* {{.Body}} [#{{.Custom.Issue}}](https://github.com/hashicorp/terraform-provider-null/issues/{{.Custom.Issue}})'
custom:
- key: Issue
  type: int
  minInt: 1
```